### PR TITLE
Update python-quiz.md

### PR DESCRIPTION
**Pull request by tripathisharad**
Hi I have recently encountered with this question and I have not found the solution in above code hence I have added it

### CODE REVIEW

**Review by GTekSD — APPROVED** [review]

File: python/python-quiz.md [hunk 1]
```diff
@@ -2099,3 +2099,11 @@ else:
...
 - [ ] 128
 - [x] No fixed length is specified
+
+#### Q153.The process of pickling in Python includes?
+
+- [x] conversion of a Python object hierarchy into byte stream
+- [ ] conversion of a datatable into a list
+- [ ] conversion of a byte stream into Python object hierarchy
+- [ ] conversion of a list into a datatable
+
```
[the remark this answers is not in the document]
Comment (tripathisharad) [reply python/python-quiz.md:2109]: Hi I have explained the solution thankyou.

[the remark this answers is not in the document]
Comment (GTekSD) [reply python/python-quiz.md:2109]: Good Job!

**Review by GTekSD — APPROVED** [review]

File: python/python-quiz.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (tripathisharad) [reply python/python-quiz.md:2109]: Hello @GTekSD please merge the request with Hacktoberfest and Hacktoberfest accepted thankyou.

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
@tripathisharad pls resolve merge conflicts, thanks

**Comment by Ebazhanov** [comment]
Please @tripathisharad fix merge conflicts because someone people already making PR based on your changes: https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/pull/4726

**Pull request closed without merging**